lib/suspenders/generators/app_generator.rb

```diff
@@ -114,6 +114,7 @@ def setup_production_environment
       build :enable_rack_canonical_host
       build :enable_rack_deflater
       build :setup_asset_host
+      build :create_heroku_pipelines_config_file
```

Comment:
This will be run on the app, but will only affect staging and production environments. `suspenders_customization` seems fine to me.